be/src/vec/exec/scan/new_file_text_scanner.cpp

```diff
@@ -45,8 +45,9 @@ Status NewFileTextScanner::open(RuntimeState* state) {
 
 Status NewFileTextScanner::_get_block_impl(RuntimeState* state, Block* block, bool* eof) {
     SCOPED_TIMER(_read_timer);
-    RETURN_IF_ERROR(init_block(block));
-
+    if (!_is_load) {
```

Comment:
> No need to check `_is_load`. Because if this is for load job. the src tuple descriptor is not null, which has already been checked in `init_block()`

I was thinking, for load job, the `_input_block` has already been initialized in `VScanner::_init_input_block`. Checking `_is_load` is to avoid initialize the `_input_block` again.